runelite-client/src/main/java/net/runelite/client/plugins/runecraft/RunecraftPlugin.java

```diff
@@ -82,6 +86,12 @@
 	@Getter(AccessLevel.PACKAGE)
 	private NPC darkMage;
 
+	@Getter(AccessLevel.PACKAGE)
+	private GameObject denseRunestoneSouth;
```

Comment:
I know the other properties above and below these lines are incorrectly indented, but new code should follow the proper indentation. (feel free to fix the incorrect ones as well if you'd like)

Reply:
Never mind, it looked wrong because of my refined github view settings; looks good to me. :+1: